app/src/test/java/org/simple/clinic/analytics/UpdateAnalyticsUserIdTest.kt

```diff
@@ -61,55 +60,18 @@ class UpdateAnalyticsUserIdTest {
       loggedInStatus: User.LoggedInStatus,
       shouldSetUserId: Boolean
   ) {
+    // given
     val user = PatientMocker.loggedInUser(loggedInStatus = loggedInStatus)
     whenever(userSession.loggedInUser()).thenReturn(Observable.just(Just(user)))
 
-    updateAnalyticsUserId.listen(scheduler)
+    // when
+    updateAnalyticsUserId.listen()
 
+    // then
     if (shouldSetUserId) {
-      assertThat(reporter.userId).isEqualTo(user.uuid.toString())
+      assertThat(reporter.user).isEqualTo(user)
```

Comment:
Can we remove this conditional from the test?

Reply:
Yup, I'll try doing this.

Reply:
Done.